app/main.py

```diff
@@ -16,4 +16,17 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+types = [
+    lucky_number,
+    pi,
+    one_is_a_prime_number,
+    name, my_favourite_films,
+    profile_info,
+    marks,
+    collection_of_coins
+]
+
+sorted_variables = {
+    "mutable": [el for el in types if isinstance(el, (list, set, dict))],
+    "immutable": [el for el in types if not isinstance(el, (list, set, dict))]
```

Comment:
use more informative names for variables, not "el"

Reply:
Also it will be much easier to write variables in dict by ur hands, not using 2 loops for this ))